test/ClangImporter/objc_async.swift

```diff
@@ -381,3 +381,15 @@ public struct SomeWrapper<T: AuditedNonSendable> {
 }
 
 extension SomeWrapper: Sendable where T: Sendable {}
+
+
+// rdar://96830159
+@MainActor class SendableCompletionHandler {
+  var isolatedThing: [String] = []
+
+  func makeCall(slowServer: SlowServer) {
+    slowServer.doSomethingSlow("churn butter") { (_ : Int) in
+      let _ = self.isolatedThing // expected-warning {{main actor-isolated property 'isolatedThing' can not be referenced from a Sendable closure; this is an error in Swift 6}}
```

Comment:
Looks like the test job failed, so as long as you’re going to update this PR, you might as well put the expectation on the next line like the tests above this one do.